documentation/interpretation_guide.md

```diff
@@ -73,24 +83,32 @@ If there are restrictions for the circulation of private cars based on certain c
+- If a country’s borders are completely closed, but due to a civil war, or other non-covid related reasons, this should still be coded as a 4. While this is not a covid specific policy, in practice the border is still closed so we want to best represent the lived reality of people living there, in order to understand the effect this could have on COVID-19.
 
 
 ### E1- Income Support
 
+-  This indicator does not have a binary flag variable to reflect geographic variation in policies. This implies that this policy applies generally across the whole jurisdiction. Therefore, we generally code the level of policy that applies everywhere across the jurisdiction- the highest common value of policy nationwide or statewide. We do not code any policies of a higher value that are only available in limited areas.  *For example*  if income support is only being provided by some cities, and not across the whole country or state, this cannot be coded. It is only recorded when the support recorded applies nationally or statewide.
 - Formal sector workers are people who are employed with contracts, and pay taxes. Informal workers may be roadside vendors, work on markets, and do not have a formal contract for their work. 
 - If benefits in kind are being given (e.g. dry rations), this is recorded as a 0 with a note. It is not recorded as a non-zero value as it is not money being given or salaries replaced.
 - Edge cases and marginal examples
     - *Widening eligibility for unemployment benefit:* For example, if a nation or state had worked within the existing unemployment scheme to offer expanded eligibility during the COVID-affected period, for example giving benefits to those who wouldn’t normally be paid leave. This expands unemployment benefits to a wider group , and would therefore be recorded as the appropriate non-zero value depending on the value/coverage.
     - *Expedited access to unemployment benefit:* If a state takes policy action to make it faster and easier to access nation or  state/federal unemployment benefits during the COVID-affected period, but doesn’t expand eligibility criteria, this is only recorded as a note. The underlying policy hasn’t changed or expanded to reach more people.
 
+### E2- Debt/contract relief for households
+- This indicator does not have a binary flag variable to reflect geographic variation in policies. This implies that this policy applies generally across the whole jurisdiction. Therefore, we generally code the level of policy that applies everywhere across the jurisdiction- the highest common value of policy nationwide or statewide. We do not code any policies of a higher value that are only available in limited areas.  *For example*  if debt relief is only being provided by some cities, and not across the whole country or state, this cannot be coded. It is only recorded when the support recorded applies nationally or statewide.
```

Comment:
As elsewhere, let's get rid of second sentence, and make sure it is worded for data users (not "this cannot be coded")